### PR TITLE
SNAP-362 Updated visited link color to match normal link color

### PR DESCRIPTION
**Pull request by tparkerosi**
https://osi-cwds.atlassian.net/browse/SNAP-362
Update footer link color to match unvisited color.

## Description
Part of ADA story. Link color was updated to handle with contrast issues in a previous commit. This commit updates the visited color to match the unvisited color. This was the behavior of the link prior to change.

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [] I have ran all tests for the project.
- [ ] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by tparkerosi** [comment]
retest this please

**Comment by tparkerosi** [comment]
retest this please

File: app/assets/stylesheets/accessibility.scss [hunk 1]
```diff
@@ -32,6 +32,6 @@
   transition: all .2s ease-in-out;
 }
 
-.footer a {
+.footer a, .footer a:visited{
```
Comment (wmitchell-cambria) [inline app/assets/stylesheets/accessibility.scss]: Space before the bracket...probably failing lint?

**Comment by tparkerosi** [comment]
retest this please

**Comment by tparkerosi** [comment]
retest this please

**Comment by pokharelmanish** [comment]
retest this please

**Pull request merged**